theme6/source/categories.html.erb

```diff
@@ -305,12 +305,10 @@
   </div>
 
   <div class="scroll-up">
-    <a href="#">
-      <button type="button" class="scroll-up-button" name="scrollup">
-        <h3>SCROLL UP</h3>
-        <img class="up-arrow" src="/images/arrow_up.png" alt="Arrow Uṕ">
-      </button>
+    <a href="#" class="scroll-up-button" name="scrollup">
```

Comment:
This should be a button... or was it an specific purpose of having a button wrapped by an anchor?

Reply:
We used the `a` to navigate to the top of the page, I remember that one of your comments was that anchors are use to nagivate to different part of the page and buttons to perform actions. Is it correct?

Reply:
Hell yeah, that's true 🤓, good answer